buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -53,19 +51,21 @@ java {
 
 tasks.withType<JavaCompile>().configureEach {
   with(options) {
-    release.set(minJavaVersionSupported.majorVersion.toInt())
+    release.set(otelJava.minJavaVersionSupported.map { it.majorVersion.toInt() })
```

Comment:
why `map` and not `get`?

Reply:
Not in `afterEvaluate` - where possible we want to map without relying on evaluation order which results in less fragility in build scripts. Because of Gradle legacy it's unfortunately not always possible though so is a bit of a lost cause in some sense...